custom-http/build.gradle

```diff
@@ -1,8 +1,8 @@
 plugins {
     id 'com.android.library'
     id 'kotlin-android'
+    id 'maven-publish'
```

Comment:
I believe it should be in publish_local.gradle file. 
is there any specific reason for adding it here? :)

Reply:
We are publishing the whole module so all plugins should be in one place

Reply:
I suggest to keep it in the publish_local.gradle file and use it while publishing any other modules also.